xbmc/LangInfo.cpp

```diff
@@ -1533,3 +1540,13 @@ void CLangInfo::AddLanguages(std::vector<StringSettingOption> &list)
   for (const auto& language : languages)
     list.emplace_back(language, language);
 }
+
+namespace
+{
+std::string GetDateStringWithFormat(const CDateTime& date, const std::string& format)
+{
+  // Return the formatted date together with the format used.
+  // eg: '1/02/2003 (D/MM/YYYY)'
+  return date.GetAsLocalizedDate(format)  + " (" + format + ")";
+}
+}
```

Comment:
Please put this at the beginning of the source file. Then you don't need to forward declare this function.

Reply:
@ksooo - Thanks for your response.

The function has been moved to the beginning of the source file and the forward declaration has been removed.  Jenkins formatting has also been applied.